lab01-activity.Rmd

```diff
@@ -273,7 +279,7 @@ At the end of every lab, replace TODO with DONE to the left of each task to indi
 |status|task|
 |-|-|
 |DONE|This file has been knit|
-|TODO| My .bashrc file exists|
-|TODO| Miniconda is installed|
-|TODO| I was able to install both Bioconda and samtools using conda|
+|DONE| My .bashrc file exists|
+|DONE| Miniconda is installed|
+|DONE| I was able to install both Bioconda and samtools using conda|
 |TODO| The last few code chunks that run Python code run without errors when I knit this document|
```

Comment:
I cannot see any problem with your code, here. But, if you still have problem with knitting the document please let me know.